app/build.gradle

```diff
@@ -99,6 +96,8 @@ dependencies {
     implementation "com.android.support:preference-v14:$androidSupportVersion"
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
     implementation 'com.google.guava:guava:23.2-android'
+    implementation 'com.google.zxing:core:3.2.1'
+    implementation 'com.journeyapps:zxing-android-embedded:3.2.0@aar'
```

Comment:
We need these to generate the QR Code right?

Reply:
I've just investigated this and found that we actually do not need these two lines, since the journeyapps.zxing is already included a view lines below that, so I'll remove those.

Reply:
PR: #989